Assets/Scripts/InteractableController.cs

```diff
@@ -40,13 +45,32 @@ void OnTriggerExit2D(Collider2D other)
     // Update is called once per frame
     void Update()
     {
+        //Debug.LogFormat("Text number is {0}", textnumber);
+        textnumber = Random.Range(1, 5);
+
         if (playerIsColliding)
         {
             if (Input.GetKeyDown(KeyCode.Z) || Input.GetKeyDown("space")) 
             {
                 HandleInput();
             }
-        } 
+        }
+        if (textnumber is 1)
```

Comment:
Another bonus if we refactor all of our text options into an array -- we can make assigning the value to `text` much easier! like this, kinda:

```c#
text = textArray[textNumber]
```

I don't know C# array syntax super well off the top of my head so I'd have to look into this, but it'd save us, like, 15 lines. :) Let's not worry for now, but feel free to file an issue to optimize this down the road and I'll tackle it when I have some time.

Reply:
This makes enough sense to me and I'm into it. I filed an issue and assigned it to you. Feel free to edit the issue if I didn't explain it exactly right.